Contest_Archive_Format.md

```diff
@@ -16,708 +17,151 @@ disk, including:
     tools or for teams to compete against live data
   - As a base for offline analysis
 
-This standard lays out the relative location and format of each type of
-contest-related information when reading or writing to disk. The top
-level structure is inspired by the [Contest API](contest_api) structure.
-
 ## Archive contents
 
 The package consists of a single directory containing files as described
 below, or alternatively, a ZIP compressed archive of the same files. It is
```

Comment:
Similar to the problem package format: good to clarify that the zip file also includes that directory (or not?).

Reply:
Clarified that the directory *is* included.